README.md

```diff
@@ -6,13 +6,11 @@ directly to [Dynatrace](https://www.dynatrace.com).
 More information on exporting OpenTelemetry metrics to Dynatrace can be found in the
 [Dynatrace documentation](https://www.dynatrace.com/support/help/extend-dynatrace/opentelemetry/opentelemetry-metrics).
 
-This exporter is built against the OpenTelemetry .NET
-SDK [v1.2.0](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.2.0) which is the first stable
-release for the metrics SDK and should be compatible with this and later versions.
+This exporter is built against the OpenTelemetry .NET SDK [v1.3.2](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.3.2). It should be compatible with applications targeting OpenTelemetry .NET SDK version 1.2.0 and higher.
```

Comment:
Is it backwards compatible with 1.2.0?

Reply:
Fixed. I removed the sentence that this "should be compatible with later versions".

To be compatible with lower versions, we would need to target a lower version here. So, for ex, if we wanted to support 1.2.x 1.3.x.. we would need to target here `1.0.1`. Floating numbers in nuget packages (`1.*`) doesn't work as "expected" for NuGet packages. ([More info](https://stackoverflow.com/questions/56873251/correct-way-to-distribute-nuget-package-with-floating-version)). But I don't think we should do that, as those versions have well defined problems. We didn't have any complaint so far with our current versioning so I assume people keep the versions close together.

Reply:
Thanks!